### ISSUE TITLE
Unused function arguments

### ISSUE THREAD

**Issue by AdrianSosic** [jsh9/pydoclint#87]
Hi 👋 I wonder if it's possible to somehow cleverly handle the case where there are unused function arguments that don't require a docstring. These situations may occur when you are forced to maintain a specific function signature, e.g. when writing [attrs-validators](https://www.attrs.org/en/stable/examples.html#validators).

For example:
```python
def foo(_: int, b: float):
    """Bar.

    Args:
        b: a number
    """
```

In this case, pydoclint will complain with
```
debugging.py:1:1: DOC101 Function `foo`: Docstring contains fewer arguments than in function signature. 
debugging.py:1:1: DOC103 Function `foo`: Docstring arguments are different from function arguments. (Or could be other formatting issues: https://jsh9.github.io/pydoclint/violation_codes.html#notes-on-doc103 ). Arguments in the function signature but not in the docstring: [_: int, b: float]. Arguments in the docstring but not in the function signature: [b: ].
```

**Comment by jsh9** [comment]
Hi @AdrianSosic , thanks for submitting this issue!

I've made a code change in the PR linked above.

It ignores `_`, `__`, `___`, ... in the argument list of the function signature.  Are they all the cases that I should consider, or are there anything else?

If not, I'll merge this PR and publish a new version.

**Comment by AdrianSosic** [comment]
No, these are all the cases that I can currently think of… Awesome, thanks a lot for the quick fix!! 👍

**Comment by AVHopp** [comment]
Following up on this: I work on a project where we quite a lot of attrs-validation function. All of these start with `_validate`, and we would like to include a `Raises:` section in the docstring without the need of specifying the `Args:` section of every docstring. This however raises the same errors mentioned in the original issue here.

My quetsion is thus: Is it possible to configure something like that without having to put ´#noqa` manually everywhere? If not, would including an option to exclude errors based on specific function names or decorators be a reasonable option to add?

**Comment by jsh9** [comment]
Hi @AVHopp , I think for your use case, using `#noqa` at each function may actually be the cleanest solution.

If pydoclint is to implement "custom filter", it will need to parse a config file, which will contain the file name, the function name, the violation to be ignored...  This is actually not much different from the flake8 config file.

**Comment by AVHopp** [comment]
Ok, thank you for the suggestion :)
